frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -60,25 +62,35 @@ const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
     return onClose();
   };
 
+  const handleScroll = (e: React.UIEvent<HTMLDivElement>) =>
+    setHeaderStyles(
+      classNames(
+        'co-quick-start-drawer-head',
+        e.currentTarget.scrollTop && 'pf-u-box-shadow-md-bottom',
+      ),
+    );
+
   const panelContent = quickStart ? (
-    <DrawerPanelContent>
-      <DrawerHead>
-        <div className="co-quick-start-drawer__title">
-          <Title
-            headingLevel="h1"
-            size="xl"
-            style={{ marginRight: 'var(--pf-global--spacer--md)' }}
-          >
-            {quickStart?.spec.displayName}
-          </Title>
-          <Title headingLevel="h6" size="md" className="text-secondary">
-            {`${quickStart?.spec.duration} minutes`}
-          </Title>
-        </div>
-        <DrawerActions>
-          <DrawerCloseButton onClick={handleClose} />
-        </DrawerActions>
-      </DrawerHead>
+    <DrawerPanelContent onScroll={handleScroll}>
+      <div className={headerStyles}>
+        <DrawerHead>
```

Comment:
can we add the header styles to DrawerHead directly instead of wrapping them into a new div.

Reply:
The `DrawerHead` component passes `className` to it's child, so only the title text gets the shadow applied.